corehq/apps/userreports/templates/userreports/edit_data_source.html

```diff
@@ -30,36 +30,49 @@
         <button type="submit" class="btn btn-default disable-on-submit" {% if data_source.disable_destructive_rebuild %}disabled{% endif %}>
           {% trans 'Rebuild Data Source'%}
         </button>
-        {% if not data_source.disable_destructive_rebuild %}
-        <span class="hq-help-template"
-          data-title="{% trans_html_attr "Rebuild Unavailable" %}"
-          data-content="{% trans_html_attr "Fully rebuilding has been disabled for this data source. Please use the in place rebuild if necessary." %}"
-          data-placement="left"></span>
+        {% if data_source.disable_destructive_rebuild %}
+          <span class="hq-help-template"
+            data-title="{% trans_html_attr "Rebuild Unavailable" %}"
+            data-content="{% trans_html_attr "Fully rebuilding has been disabled for this data source. Please use the in place rebuild if necessary." %}"
+            data-placement="left"></span>
         {% endif %}
       </div>
-      <a href="{% url 'preview_configurable_data_source' domain data_source.get_id %}" class="btn btn-default">{% trans 'Preview data' %}</a>
-      <a href="{% url 'summary_configurable_data_source' domain data_source.get_id %}" class="btn btn-default">{% trans 'Data Source Summary' %}</a>
-      <a href="{% url 'configurable_data_source_json' domain data_source.get_id %}"
-         class="btn btn-default track-usage-link"
-         data-category="UCR"
-         data-action="View Source"
-         data-label="Data Source">{% trans 'Data Source Source (Advanced)' %}</a>
     </form>
-    {% if not data_source.is_deactivated %}
-      <hr />
-      <form method='post' action="{% url 'resume_build' domain data_source.get_id %}">
-        {% csrf_token %}
-        <button type="submit" class="btn btn-default disable-on-submit">
-          {% trans 'Resume Build (Advanced)' %}
-        </button>
-      </form>
-      <form method='post' action="{% url 'build_in_place' domain data_source.get_id %}">
+    <a href="{% url 'preview_configurable_data_source' domain data_source.get_id %}" class="btn btn-default">{% trans 'Preview data' %}</a>
+    <a href="{% url 'summary_configurable_data_source' domain data_source.get_id %}" class="btn btn-default">{% trans 'Data Source Summary' %}</a>
+    <div class="btn-group">
+      <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        {% trans "Advanced" %} <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu">
+        <li>
+          <a href="{% url 'configurable_data_source_json' domain data_source.get_id %}"
+             class="track-usage-link"
+             data-category="UCR"
+             data-action="View Source"
+             data-label="Data Source">{% trans "Data Source Source" %}</a>
```

Comment:
unrelated to PR: Just realized how ridiculous "source source" sounds. would that be changed to "Data Source JSON" or something? or do we have a convention for this sort of text?

Reply:
Agree it sounds silly, I can change it in the next PR (I'm basically timeboxing 30 min or so a day for front end changes). I don't think there's a convention since in general displaying json is not a thing we should do. "Data Source JSON" is probably appropriately intimidating.